src/screens/LinkContextScreen/elements/Header.js

```diff
@@ -241,19 +241,19 @@ const styles = StyleSheet.create({
     marginLeft: -4,
   },
   wrapperText: {
-    backgroundColor: 'white',
+    backgroundColor: COLORS.white,
     borderRadius: 8,
-    borderColor: '#00ADB5',
+    borderColor: COLORS.holyTosca,
     width: 28,
     height: 28,
     justifyContent: 'center',
     alignItems: 'center',
     borderWidth: 0.5,
   },
   wrapperTextUnFollow: {
-    backgroundColor: '#00ADB5',
+    backgroundColor: COLORS.holyTosca,
     borderRadius: 8,
-    borderColor: '#00ADB5',
+    borderColor: COLORS.holyTosca,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The borderColor and backgroundColor for the `wrapperTextUnFollow` style are the same, which makes the border invisible. If this is intentional, it's fine. However, if you want the border to be visible, consider using a different color for the border.
```diff
   wrapperTextUnFollow: {
    backgroundColor: COLORS.holyTosca,
     borderRadius: 8,
-256:     borderColor: COLORS.holyTosca,
+256:     borderColor: COLORS.someOtherColor,
     width: 28,
     height: 28,
     justifyContent: 'center',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->